TypeCobol.LanguageServer/TypeCobolServer.cs

```diff
@@ -172,22 +171,9 @@ protected void MissingCopiesDetected(TextDocumentIdentifier textDocument, List<s
                 var missingCopiesParam = new MissingCopiesParams();
                 missingCopiesParam.textDocument = textDocument;
 
-#if EUROINFO_RULES
```

Comment:
Can you put back this code and separate CPY and non CPY for EUROINFO_RULES ?
We will certainly need it on the client side.